docs/rest_api.md

```diff
@@ -1255,6 +1255,22 @@ configurations.
 
 ### The `hil.ext.auth.database` auth backend
 
+#### list_user
+
+`PUT /auth/basic/user`
+
+Return a list of all users in HIL
```

Comment:
It's a bit weird to word it this way, since you're not actuall returning a list, but an object mapping user names -> info. It might also be more informative to add more than one user to the sample output below.

Reply:
@zenhack make sense. Changing it to `List all users` at the moment. Feel free to let me know if you have a better way of describing it.